go-controller/vendor/github.com/coreos/go-iptables/iptables/iptables.go

```diff
@@ -491,6 +578,57 @@ func (ipt *IPTables) run(args ...string) error {
 	return ipt.runWithOutput(args, nil)
 }
 
+// runWithOutput runs an iptables command with the given arguments,
+// writing any stdout output to the given writer
```

Comment:
nit: change doc string to expand more what this does and why its different from runWithOutput

Reply:
not my change